tests/login-register-logout-smoke.spec.js

```diff
@@ -0,0 +1,27 @@
+const chai = require("chai");
+const expect = chai.expect;
```

Comment:
You can combine this 2 lines into 1: const { expect } = require('chai') or const expect = require('chai').expect.